apollo-router/src/plugins/cache/cache_control.rs

```diff
@@ -154,16 +160,23 @@ impl CacheControl {
     pub(crate) fn to_headers(&self, headers: &mut HeaderMap) -> Result<(), BoxError> {
         let mut s = String::new();
         let mut prev = false;
+        let now = now_epoch_seconds();
         if let Some(max_age) = self.max_age {
-            write!(&mut s, "{}max-age={}", if prev { "," } else { "" }, max_age)?;
+            //FIXME: write no-store if max_age = 0?
```

Comment:
Does it need to be fixed at part of this PR ?

Reply:
yes, because the `max-age` value returned to the client was incorrect